### ISSUE TITLE
Cannot send invoice

### ISSUE THREAD

**Issue by davwheat** [crater-invoice/crater#775]
**Describe the bug**
When hitting the email preview API endpoint, errors are returned stating that several fields are required, but these are provided in the query params in the request.

**Expected behavior**
Preview functionality should work as expected without errors.

**Screenshots**

![image](https://user-images.githubusercontent.com/7406822/152667729-32d55285-8b53-4116-8dd9-01d42e438312.png)

![image](https://user-images.githubusercontent.com/7406822/152667743-b3938577-1a8c-472b-aad3-2fd5fc1328c7.png)

**Please complete the following information:**
- Crater version: 6.0.2
- PHP version: 8.1.2
- Database type and version: MySQL 8.0.27-0ubuntu0.21.04.1

**Optional info**
- OS:  [e.g. Ubuntu] Ubuntu Server 21.03
- Browser: [e.g. chrome, safari] Chrome

**Comment by davwheat** [comment]
This is still an issue on 6.0.4. I cannot send any invoice emails to clients.

**Comment by mohitpanjwani** [comment]
Hey @davwheat, not sure what's wrong here. 

I cannot seem to reproduce this one. You're really the first one to report this issue 🤔

**Comment by davwheat** [comment]
Are there any steps I could take to provide more detailed debugging info?

My setup is mildly complex. (Will add details in a moment.)

**Comment by mohitpanjwani** [comment]
It seems like the get parameters are not being passed to the controller while POST requests seem to be working correctly for you. 

Yeah this is definitely has something to do with your configuration.

**Comment by davwheat** [comment]
The query string params are making their way as far as the Nginx instance running Crater:

![](https://u.davwheat.dev/XdKPuMJz.png)

**Comment by mohitpanjwani** [comment]
> The query string params are making their way as far as the Nginx instance running Crater at least:
> 
> ![](https://camo.githubusercontent.com/4c459e27aa2195e06e6c29eacf98501f3135b032e51f78c9f0090fa3255b8331/68747470733a2f2f752e64617677686561742e6465762f58644b50754d4a7a2e706e67)
> 
> I can also reproduce your suggestion of the issue by filtering the invoice list to "SENT" only and it shows all invoices.

Yes, maybe check the nginx configuration to make sure that your rewrite rules don't ignore the query params? 

Crater wouldn't function properly without those. You'd have all sorts of issues on filters, etc.

**Comment by davwheat** [comment]
Yeah, found the issue. A stray nginx config file I created was calling `try_files` with `index.php` without any query string. All works fine now, sorry for the trouble caused!
